catalog/models.py

```diff
@@ -95,6 +95,7 @@ class ProductQuerySet(models.QuerySet):
     def get_offset(self, start, step):
         return self[start:start + step]
 
+    # @todo #183:60m Try to remove `ordering` arg from ProductQuerySet.get_by_category
```

Comment:
it's about 30 min i think. We should move it from model to context.
And use ready context arch for it